application/event_replay/event_replay.py

```diff
@@ -32,8 +32,10 @@ def lambda_handler(event: Dict[str, Any], context: LambdaContext) -> str:
     logger.set_correlation_id(correlation_id)
     validate_event(event)
     odscode = event["odscode"]
+    org_type = event["org_type"]
```

Comment:
Are you expecting the user to pass in the org_type_id?

Reply:
If so there needs to be documentation changes on confluence. Or you get the organisation type id from the change event in dynamodb

Reply:
No, I am expecting org_type from the event object